packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -93,6 +94,17 @@ export async function handleSending(
     deps.telemetry.recordHistogram('ilp_pay_time_ms', payDuration, {
       description: 'Time to complete an ILP payment'
     })
+
+    await deps.telemetry.incrementCounterWithTransactionFeeAmount(
```

Comment:
maybe something like `incrementCounterWithTransactionAmountDifference` is more generic? since I feel like telemetry service shouldn't necessarily know what a "fee" is, just the caller.

or we pull the meat of the function into this file instead (the conversion of the two amounts), and just call the existing `incrementCounterWithTransactionAmount` at the end.